src/components/Medicine/MedicationRequestTable/index.tsx

```diff
@@ -167,17 +171,23 @@ export default function MedicationRequestTable({
                     </Link>
                   </Button>
                   <Button
-                    asChild
                     variant="outline"
+                    disabled={
+                      !(
+                        activeMedications?.results?.length ||
+                        stoppedMedications?.results?.length
+                      )
+                    }
+                    onClick={() =>
```

Comment:
Let's stick with using Link component inside Button as it maintains semantic HTML +  accessibility.

There's a few other instances in the code base that need to be adjusted as well, but we can take that up later.